imcsdk/apis/server/adaptor.py

```diff
@@ -85,41 +145,37 @@ def setup_vic_adaptor_properties(handle, adaptor_slot=1, fip_mode=None,
     return mo
 
 
-def get_vnic(handle, adaptor_slot, name):
+def get_vnic(handle, adaptor_slot, name, server_id=1, **kwargs):
     """
     This method is used to get a vnic
     Args:
         handle (ImcHandle)
-        adaptor_slot (int): PCI slot number of the adaptor
+        adaptor_slot (string): PCI slot number of the adaptor
         name (string): Name for the vnic to be deleted
+        server_id (int): Server Id for C3x60 platforms
+        kwargs: key=value paired arguments
 
     Returns:
         AdaptorHostEthIf object
     """
 
-    from imcsdk.imcexception import ImcOperationError
     from imcsdk.mometa.adaptor.AdaptorHostEthIf import AdaptorHostEthIf
 
-    dn = "sys/rack-unit-1/adaptor-" + str(adaptor_slot)
-    mo = handle.query_dn(dn)
-
-    if mo is None:
-        raise ImcOperationError("Get Vnic",
-                                "Adaptor is not available")
-
-    vnic_mo = AdaptorHostEthIf(parent_mo_or_dn=dn, name=name)
+    mo = _get_adaptor(handle, adaptor_slot, server_id, **kwargs)
+    vnic_mo = AdaptorHostEthIf(parent_mo_or_dn=mo.dn, name=name)
     vnic_mo = handle.query_dn(vnic_mo.dn)
 
     return vnic_mo
 
 
 def create_vnic(handle, adaptor_slot, name, channel_number, mac, mtu, cos="",
-                port_profile="", pxe_boot=False, uplink_port=0):
+                port_profile="", pxe_boot=False, uplink_port=0,
```

Comment:
I think the UI has a default value for `mtu`, probably `1500`. If so, please specify a default value